apps/desktop/src/lib/commit/CommitMessageInput.svelte

```diff
@@ -249,16 +272,12 @@
 			/>
 		{/if}
 
-		{#if title.length > 1}
-			<Tooltip
-				text={title.length > maxTitleLength
-					? `Summary is too long,\n${maxTitleLength} characters or less is best.\nUse description for more details`
-					: `Summary chars`}
-			>
-				<span class="text-11 text-semibold text-body commit-box__textarea-char-counter-label">
-					{title.length}
-				</span>
-			</Tooltip>
+		{#if title.length > 0 && isTitleFocused}
+			{@render charCounter('Summary chars', title.length)}
+		{/if}
+
+		{#if description.length > 0 && isDescriptionFocused}
+			{@render charCounter('Description chars', description.length)}
```

Comment:
Should this display the character count for the description line that you're currently on, rather than the entire description character count?

Reply:
Understand. However, this will be more complex because we need to dynamically determine where the line ends and the current position of the text caret, as it's a single textarea field.

Reply:
In V3, we’ll have an auto-wrap option. For now, I’m not sure if it’s worth investing more time in this—unless you already have a working solution. In that case, I’d be happy to copy/paste it! 😆